docs/user/cli-reference.md

```
@@ -1297,15 +1297,15 @@ Results are output in CTRF (Common Test Report Format) — an industry-standard
 | `passed` | Check or constraint passed |
 | `failed` | Check or constraint failed |
 | `skipped` | Check could not be evaluated (missing data, no-cluster mode) |
-| `other` | Unexpected outcome (crash, OOM, timeout) |
+| `other` | Indeterminate outcome — the check produced no usable verdict (crash, OOM, or a validator Job that failed for a non-deadline reason with no inspectable pod) |
 
 **Exit Codes:**
 | Code | Description |
```

markdownlint-cli2 (0.23.2)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)